### PR TITLE
Adding 2016H relvals [81X]

### PR DESCRIPTION
**Pull request by fabozzi**
2016H relvals have been added 
The RECO step uses a Prompt-like GT provided by Alca team (thanks!)

@mmusich @franzoni @arunhep

### CODE REVIEW

**Comment by fabozzi** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_standard.py [hunk 1]
```diff
@@ -205,6 +205,19 @@
...
+workflows[136.770] = ['',['RunDoubleEG2016H','HLTDR2_2016','RECODR2_2016reHLT_skimDoubleEG_HIPM_Prompt','HARVESTDR2']]
+workflows[136.771] = ['',['RunDoubleMuon2016H','HLTDR2_2016','RECODR2_2016reHLT_HIPM_Prompt','HARVESTDR2']]
+workflows[136.772] = ['',['RunJetHT2016H','HLTDR2_2016','RECODR2_2016reHLT_skimJetHT_HIPM_Prompt','HARVESTDR2']]
+workflows[136.773] = ['',['RunMET2016H','HLTDR2_2016','RECODR2_2016reHLT_skimMET_HIPM_Prompt','HARVESTDR2']]
+workflows[136.774] = ['',['RunMuonEG2016H','HLTDR2_2016','RECODR2_2016reHLT_skimMuonEG_HIPM_Prompt','HARVESTDR2']]
+workflows[136.775] = ['',['RunSingleEl2016H','HLTDR2_2016','RECODR2_2016reHLT_HIPM_Prompt','HARVESTDR2']]
+workflows[136.776] = ['',['RunSingleMu2016H','HLTDR2_2016','RECODR2_2016reHLT_HIPM_Prompt','HARVESTDR2']]
+workflows[136.777] = ['',['RunSinglePh2016H','HLTDR2_2016','RECODR2_2016reHLT_skimSinglePh_HIPM_Prompt','HARVESTDR2']]
+workflows[136.778] = ['',['RunZeroBias2016H','HLTDR2_2016','RECODR2_2016reHLT_HIPM_Prompt','HARVESTDR2']]
+workflows[136.779] = ['',['RunMuOnia2016H','HLTDR2_2016','RECODR2_2016reHLT_skimMuOnia_HIPM_Prompt','HARVESTDR2']]
```
Comment (makortel) [inline Configuration/PyReleaseValidation/python/relval_standard.py]: Are you sure you want "HIP mitigation" for 2016H?

Comment (fabozzi) [inline Configuration/PyReleaseValidation/python/relval_standard.py]: Hi Matti, good point! This is not needed.

**Comment by fabozzi** [comment]
please test

**Comment by mmusich** [comment]
@fabozzi in case you want to place an analogous PR in 90X as well please use:

```
+# GlobalTag for Run2 data 2016H relvals only: Prompt Conditions + fixed L1 menu (to be removed)  
+    'run2_data_promptlike' : '90X_dataRun2_PromptLike_v0',
```

diff is available [here](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/81X_dataRun2_PromptLike_v0/90X_dataRun2_PromptLike_v0)

**Comment by davidlange6** [comment]
indeed, this is needed in 90x before elsewhere (and being in the 900pre1 relval campaign for ctpps is already interesting)

**Comment by fabozzi** [comment]
+1

**Comment by mmusich** [comment]
+1

**Comment by fabozzi** [comment]
I have fixed input dataset name for DoubleEG.
Also, relvals test has been submitted in 900_pre1

**Comment by fabozzi** [comment]
please test

**Comment by mmusich** [comment]
+1

**Comment by fabozzi** [comment]
The relval test in 900_pre1 was OK

**Comment by fabozzi** [comment]
+1

**Pull request closed without merging**